src/main/java/com/yelp/nrtsearch/server/state/BackendGlobalState.java

```diff
@@ -96,6 +97,19 @@ public static String getUniqueIndexName(String indexName, String id) {
     return indexName + "-" + id;
   }
 
+  /**
+   * Return the base index name from the unique index identifier.
+   *
+   * @param uniqueIndexName index identifier
+   * @return indexName index name
+   * @throws NullPointerException if uniqueIndexName is null
+   */
+  public static String getBaseIndexName(String uniqueIndexName) {
+    Objects.requireNonNull(uniqueIndexName);
+    // unique identifier doesn't contain "-"
```

Comment:
Not sure what is meant by this comment since the unique index identifier does contain hyphen.

Reply:
Ah I guess you meant the yyyyMMdd part. Would be good to clarify here.